From 50c8980adb2b3661aaa66fc564e79778c5a4cf82 Mon Sep 17 00:00:00 2001
From: Friedemann Kleint <Friedemann.Kleint@digia.com>
Date: Fri, 9 May 2014 10:49:48 +0200
Subject: [PATCH] windployqt: Fix cross-compilation.

Fix signature of readPeExecutable().

Task-number: QTBUG-38160
Task-number: QTBUG-38886
Change-Id: I3ff88d502e0f9f7912b4c0741667a8f4ff71dbba
---
 src/windeployqt/utils.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/windeployqt/utils.cpp b/src/windeployqt/utils.cpp
index 8d92e62..9faa808 100644
--- a/qttools/src/windeployqt/utils.cpp
+++ b/qttools/src/windeployqt/utils.cpp
@@ -872,7 +872,7 @@ QString findD3dCompiler(Platform platform, const QString &qtBinDir, unsigned wor
 #else // Q_OS_WIN
 
 bool readPeExecutable(const QString &, QString *errorMessage,
-                      QStringList *, unsigned *, bool *)
+                      QStringList *, unsigned *, bool *, bool)
 {
     *errorMessage = QStringLiteral("Not implemented.");
     return false;
-- 
1.8.4

